macros/schema_tests/equal_rowcount.sql

```diff
@@ -17,7 +17,7 @@ b as (
     select count(*) as count_b from {{ compare_model }}
 
 ),
-final as (
+finale as (
```

Comment:
heh, let's name this `final_count` — `finale` doesn't quite feel right.

Reply:
sure, no problem